app/lib/widgets/apple_sign_in_button.dart

```diff
@@ -72,6 +72,7 @@ class SignInWithAppleButton extends StatelessWidget {
                   child: Icon(
                     FontAwesomeIcons.apple,
                     size: 26,
+                    color: Colors.white,
```

Comment:
As you can see in your last screen shot the icon was not completely white. I didn't find see this for the other icons in the app though.